assessment/Assessment-form.md

```diff
@@ -0,0 +1,92 @@
+
+# Assessment form for the Scipy tutorial: preliminar questions
+
+### What is the primary difference between geographic and projected coordinate reference systems?
+
+- Geographic coordinate reference systems use latitude and longitude while projected coordinate reference systems use XY coordinates.
+- Geographic coordinate reference systems are three-dimensional while projected coordinate reference systems are two-dimensional.
+- Projected coordinate reference systems can only be used in the southern hemisphere.
+- Geographic coordinate reference systems use the prime meridian as the origin while projected coordinate reference systems use the equator.
+
+### What is the reference line for latitude in the geographic coordinate system?
+
+- The prime meridian
+- The equator
+- The North pole
+- The South pole
+
+### Which statement is true about lines of longitude?
+
+- They run parallel to the equator.
+- They are assigned positive values in the southern hemisphere.
+- They converge at the poles.
+- The distance between lines of longitude is the same at all latitudes.
+
+### What is the origin point for the Universal Transverse Mercator (UTM) coordinate reference system?
+
+- The North pole
+- The South pole
+- The equator at a specific Longitude
+- Greenwich England
+
+### Which file is mandatory for a shapefile to represent spatial vector data?
+
+- .prj
+- .xml
+- .shp
+- .cpg
+
+### What type of data does GeoJSON format encode?
+
+- Raster data
+- Vector data
+- Both raster and vector data
+- Metadata for geospatial images
+
+---
+
+### NASA EarthData
+
+**Have you accessed NASA EarthData Cloud before this tutorial?**
+
+- Yes
+- No
+
+**How difficult do you find the process of accessing data from NASA EarthData Cloud?**
+
+- Very easy: I could access and retrieve data without any issues.
+- Somewhat easy: I could access data but I encountered minor difficulties.
+- Challenging: I encountered significant difficulties or was unable to access the data.
+
+**Have you used NASA EarthData products in your research or projects before this tutorial?**
+
+- Yes
+- No
+
+**Do you feel confident in extracting and processing data from NASA EarthData Cloud after this tutorial?**
```

Comment:
After this tutorial, how confident do you feel about extracting and processing data from NASA EarthData Cloud?